dns-policy-operator/requirements.txt

```diff
@@ -0,0 +1,3 @@
+ops==2.17
```

Comment:
Maybe you want to use the latest version 2.19.0 of ops already :)

Reply:
Fixed